README.md

```diff
@@ -1,7 +1,14 @@
 # MongoDB Atlas Service Broker
 
-Implementation of the [Open Service Broker API](https://www.openservicebrokerapi.org/) for [MongoDB Atlas](https://www.mongodb.com/cloud/atlas). Deploy this service to easily manage Atlas instances!
+Use the Atlas Service Broker to connect [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) to any platform which supports the [Open Service Broker API](https://www.openservicebrokerapi.org/), such as [Kubernetes](https://kubernetes.io/) and [Pivotal Cloud Foundry](https://pivotal.io/platform).
 
+- Provision managed MongoDB clusters on Atlas directly from your platform of choice. Includes support for all cluster configuration settings and cloud providers available on Atlas.
+- Manage and scale clusters without leaving your platform.
+- Create bindings to allow your applications access to clusters.
+
+## Documentation
+
+For instructions on how to install and use the MongoDB Atlas Service Broker please refer to the [documentation](https://docs.mongodb.com/atlas-open-service-broker).
```

Comment:
404 atm - is this definitely where they'll be?

Reply:
According to Melissa, yes. Should we remove this section and add it in again once the docs are published?

Reply:
nah, it's fine